system-tests/005/__snapshots__/005.test.js.snap

```diff
@@ -11765,169 +11765,169 @@ Array [
         "text": "Expected line length to be no more than 20 characters (max-line-length)",
       },
       Object {
-        "column": 5,
-        "line": 146,
+        "column": 25,
+        "line": 145,
```

Comment:
Trying to wrap my head around this and it's not going so well 😏 

Currently in `master` before changes by this PR we have:

https://github.com/stylelint/stylelint/blob/master/system-tests/005/stylesheet.css#L146
> `b a {}`  <- 6 characters in length

https://github.com/stylelint/stylelint/blob/master/system-tests/005/__snapshots__/005.test.js.snap#L11767-L11773
```
      Object {
        "column": 5,
        "line": 146,
        "rule": "max-line-length",
        "severity": "error",
        "text": "Expected line length to be no more than 20 characters (max-line-length)",
      },
```

Once applying the changes in this PR the above warning for line 146 is no longer reported in the _test snapshot_ but a new one is, line 149 with `.bar,` is now reported in the  test snapshot whereas it wasn't previously.

It appears that there are numerous cases of the above, the only sane piece of data I've been able to extract thus far is that both before and after this PR there are a total of 84 `max-line-length` warnings, when i manually remove all lines from the `stylesheet.css` file less than 20 characters in length I indeed have 84 lines greater than 20 characters which correlates.

Line numbers, are at best a _crap shoot_ 🎲 

If anyone else in @stylelint/core can make any sense of this it'd be appreciated 😄